### PR TITLE
DESCW-1653 addition of Creating Pages and Editor documentation

### PR DESCRIPTION
**Pull request by deakin-chris**
Addition of creating pages and using the html editor function in the Block theme. Added 'Home' to top admin bar.

### CODE REVIEW

File: documentation/docs/wp-creating-pages.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+- About Us 
+- Board of Directors
+- Company Culture
+- Press
+- Careers
+- Locations
+- Services 
+- Web Design
+- Web Development
+- Stationery Design
```
Comment (mhaswell-bcgov) [inline documentation/docs/wp-creating-pages.md:60]:
```suggestion
- About Us
	- Board of Directors
	- Company Culture
	- Press
	- Careers
	- Locations
- Services
	- Web Design
	- Web Development
	- Stationery Design
```
From the paragraph above about the structure of the pages it sounds like there should be a hierarchy that isn't coming through in the markdown output. This should fix it if I understand the hierarchy right.

Comment (deakin-chris) [reply documentation/docs/wp-creating-pages.md:60]: Good catch, thank you.

File: documentation/docs/wp-creating-pages.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+3. In the sidebar settings for the page, under Page Attributes, open the Parent Page drop-down menu. This will list all the Pages already created in your site.
+4. Select the desired page from the drop-down menu that you want to be the parent page for the current you are creating. The current page you are creating now becomes the sub page for the parent page you choose.
+5. You can change the order that your pages are displayed when using a default menu, by using the Order field of the Page Attributes module. Put the number 1 in the box for Order. This tells WordPress to display this page first on your site.
+6. Add a title for the sub page.
+7. Add body content for the sub page.
+8. Click Publish when ready.
+
+<img src="./md-assets/pages-edit-title-3.png" hspace="30">
+
+Repeat the process for your other sub pages you want to be disabled under a Parent page, but use higher numbers for the Order field: 2, 3, etc. This tells WordPress to display these pages second and third on your site.
```
Comment (mhaswell-bcgov) [inline documentation/docs/wp-creating-pages.md:93]: Is "disabled" the right word here?

**Review by mhaswell-bcgov — COMMENTED** [review]
Looks good! A lot of useful information here. I'll let @ShawnTurple look it over and approve since he knows more about this ticket than I do.

**Pull request closed without merging**